core-java-modules/core-java-string-algorithms-3/src/main/java/com/baeldung/emailvalidation/EmailValidation.java

```diff
@@ -0,0 +1,105 @@
+package com.baeldung.emailvalidation;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.commons.validator.routines.EmailValidator;
+
+public class EmailValidation {
+
+    public static boolean usingEmailValidator(String emailAddress) {
+
+        EmailValidator validator = EmailValidator.getInstance();

```

Comment:
Can't we just do this in one line?
`return EmailValidator.getInstance().isValid(emailAddress)`

I don't think we even need a helper method in this case. Let's remove it.